tests/chart_tests/test_houston_configmap.py

```diff
@@ -36,827 +36,846 @@ def common_test_cases(docs):
-    enabled."""
-    docs = render_chart(
-        values={"astronomer": {"configSyncer": {"enabled": True}}},
-        show_only=["charts/astronomer/templates/houston/houston-configmap.yaml"],
-    )
-
-    common_test_cases(docs)
-    doc = docs[0]
-    prod = yaml.safe_load(doc["data"]["production.yaml"])
-    assert prod["deployments"]["helm"]["airflow"]["webserver"]["extraVolumeMounts"] == [
-        {
-            "name": "signing-certificate",
-            "mountPath": "/etc/airflow/tls",
-            "readOnly": True,
-        }
-    ]
-    assert prod["deployments"]["helm"]["airflow"]["webserver"]["extraVolumes"] == [
-        {
-            "name": "signing-certificate",
-            "secret": {"secretName": "release-name-houston-jwt-signing-certificate"},
-        }
-    ]
-
-
-def test_houston_configmap_with_config_syncer_disabled():
-    """Validate the houston configmap and its embedded data with configSyncer
-    disabled."""
-    docs = render_chart(
-        values={"astronomer": {"configSyncer": {"enabled": False}}},
-        show_only=["charts/astronomer/templates/houston/houston-configmap.yaml"],
-    )
-
-    common_test_cases(docs)
-    doc = docs[0]
-    prod_yaml = yaml.safe_load(doc["data"]["production.yaml"])
-    assert "extraVolumeMounts" not in prod_yaml["deployments"]["helm"]["airflow"]["webserver"]
-    assert "extraVolumes" not in prod_yaml["deployments"]["helm"]["airflow"]["webserver"]
-    assert not prod_yaml["deployments"].get("loggingSidecar")
-
-
-def test_houston_configmap_with_fluentd_index_prefix_defaults():
-    """Validate the houston configmap and its embedded data with configSyncer
-    disabled."""
-    docs = render_chart(
-        values={},
-        show_only=["charts/astronomer/templates/houston/houston-configmap.yaml"],
-    )
-
-    common_test_cases(docs)
-    doc = docs[0]
-    prod_yaml = yaml.safe_load(doc["data"]["production.yaml"])
-    assert "fluentd" in prod_yaml["deployments"].get("fluentdIndexPrefix")
-
-
-def test_houston_configmap_with_fluentd_index_prefix_overrides():
-    """Validate the houston configmap and its embedded data with configSyncer
-    disabled."""
-    docs = render_chart(
-        values={"global": {"logging": {"indexNamePrefix": "astronomer"}}},
-        show_only=["charts/astronomer/templates/houston/houston-configmap.yaml"],
-    )
-
-    common_test_cases(docs)
-    doc = docs[0]
-    prod_yaml = yaml.safe_load(doc["data"]["production.yaml"])
-    assert "astronomer" in prod_yaml["deployments"].get("fluentdIndexPrefix")
-
-
-def test_houston_configmap_with_loggingsidecar_enabled():
-    """Validate the houston configmap and its embedded data with
-    loggingSidecar."""
-    docs = render_chart(
-        values={
-            "global": {
-                "loggingSidecar": {
-                    "enabled": True,
-                    "repository": "quay.io/astronomer/ap-vector",
-                    "tag": "0.22.3",
+class TestHoustonConfigmap:
```

Comment:
Why do we need this class? We're not parametrizing it. Is there some other benefit to having it?

Reply:
if you are okay with individual functions i can revert them back

Reply:
If there's a reason to use a class I'm all for it. What I observe here though is that adding the class makes the code more complicated for no benefit, and having them all as separate functions is cleaner code that is more readable.

Reply:
let me revert back those change back to previous ones